src/components/post/PostDropDown.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+import "../../styles/PostDropDown.scss";
+import Dropdown from "react-bootstrap/Dropdown";
+import editIcon from "../../images/edit-pencil.svg";
+import deleteIcon from "../../images/del-bin.svg";
+import linkIcon from "../../images/link.svg";
+
+class PostDropDown extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div>
+        <Dropdown.Item className="post-dropdown-item">
```

Comment:
I don't see`post-dropdown-item` being used in styles or in the code anywhere else. Do we even need to define the class here?

Reply:
removed